### PR TITLE
Initial CVE-2021-25657 Publication

### PR DESCRIPTION
**Pull request by AvayaSecurityAlerts**

### CODE REVIEW

File: 2021/25xxx/CVE-2021-25657.json [hunk 1]
```diff
@@ -1,18 +1,104 @@
 {
```
Comment (cve-team) [inline 2021/25xxx/CVE-2021-25657.json:1]: 'data_type' is a required property

Comment (cve-team) [inline 2021/25xxx/CVE-2021-25657.json:1]: 'data_format' is a required property

Comment (cve-team) [inline 2021/25xxx/CVE-2021-25657.json:1]: 'data_version' is a required property

Comment (cve-team) [inline 2021/25xxx/CVE-2021-25657.json:1]: 'CVE_data_meta' is a required property

Comment (cve-team) [inline 2021/25xxx/CVE-2021-25657.json:1]: 'affects' is a required property

Comment (cve-team) [inline 2021/25xxx/CVE-2021-25657.json:1]: 'problemtype' is a required property

Comment (cve-team) [inline 2021/25xxx/CVE-2021-25657.json:1]: 'references' is a required property

Comment (cve-team) [inline 2021/25xxx/CVE-2021-25657.json:1]: 'description' is a required property

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by AvayaSecurityAlerts** [comment]
Am i missing something or is https://vulnogram.github.io/cve5/ broken? I used the JSON generated by Vulnogram and it gives the format as provided by this commit.

For example, 
"dataType": "CVE_RECORD",
"dataVersion": "5.0",

and no data_format is generated in the JSON

**Comment by anthnysingleton** [comment]
Hello @AvayaSecurityAlerts,

You cannot use JSON V5 with this GitHub Pilot. You will need to use https://vulnogram.github.io/#editor to produce the JSON file in v4.

CVE Content Team

**Pull request closed without merging**